addons/sys_core/fnc_processRadioSpeaker.sqf

```diff
@@ -28,11 +28,9 @@ TRACE_2("",_unit,_playerRadios);
 private _radioId = _unit getVariable [QGVAR(currentSpeakingRadio), ""];
 if (_radioId == "") exitWith { false };
 
-// Debug #638
-if (isNil "_radioId") exitWith {
-    WARNING_1("reserved variable debug 1 - report on GitHub! [%1]",_radioId);
-    false
-};
+// Workaround #638 - _radioId is nil - TODO investigate further
+// Probable source: https://github.com/IDI-Systems/acre2/blob/c03262be2190cd54c3cb5f27d7ca3eddf7270506/addons/sys_core/fnc_remoteStartSpeaking.sqf#L107
+if (isNil "_radioId") exitWith { false };
```

Comment:
Wouldn't it be better to check `isNil` first before the `== ""` above it?

Reply:
Sure, but how rarely this happens it makes absolutely no difference.